app/packages/core/src/components/Filters/categoricalFilter/CategoricalFilter.tsx

```diff
@@ -259,6 +264,24 @@ const CategoricalFilter = <T extends V = V>({
               toKey={({ value }) => String(value)}
             />
           )}
+        {results !== null &&
+          (results.length > CHECKBOX_LIMIT || neverShowExpansion) &&
+          !skeleton &&
+          !isFilterMode && (
+            <Selector
+              useSearch={useSearch}
+              placeholder={`+ set visibility by ${name}`}
+              component={ResultComponent}
+              onSelect={onSelect}
+              inputStyle={{
+                color: theme.text.secondary,
+                fontSize: "1rem",
+                width: "100%",
+              }}
+              containerStyle={{ borderBottomColor: color, zIndex: 1000 }}
+              toKey={({ value }) => String(value)}
+            />
+          )}
```

Comment:
This looks like it is duplicated from the above but only differs in its `placeholder` value? Can the `placeholder` just prop just be changed?

```ts
placeholder={`+ ${isFilterMode ? "filter by" : "set visibility"} by ${name}`}
```